docs/troubleshooting-guide/error-message.md

```diff
@@ -0,0 +1,34 @@
+# What type of error message do you see?
+
+## API error messages
+
+- The error message appears in red text when you load a dashboard or run a question.
+- The error message contains a three-digit API error code, such as `400` or `404`.
+- If your error message contains part of your SQL query, go to SQL error messages below.
+
+### What to do:
```

Comment:
Think we can skip the "What to do " headings, and just link to where people should go.